pages/posts/w3c-wot-case-study.md

```diff
@@ -79,45 +81,47 @@ Descriptions.
 
 The ultimate metric of success for a standard like the W3C Web of Things is
 adoption, and community adoption is not something you can just "engineer".
-Because we adopt JSON Schema, developers who are familiar with it have a
-smaller learning curve when learning the W3C Web of Things standard.
+"Because we adopt JSON Schema, developers who are familiar with it have a
+smaller learning curve when learning the W3C Web of Things standard." - Ege
+Korkan
 
```

Comment:
I suggest this rewrite:
Ege Korkan commented, "Because we use JSON Schema, developers who are familiar with it have a lesser learning curve when adopting the W3C Web of Things standard."